src/adapter/properties.rs

```diff
@@ -449,6 +449,9 @@ pub(super) fn resolve_trait_property<'a, V: AsVertex<Vertex<'a>> + 'a>(
         "object_safe" => {
             resolve_property_with(contexts, field_property!(as_trait, is_dyn_compatible))
         }
+        "dyn_compatible" => {
+            resolve_property_with(contexts, field_property!(as_trait, is_dyn_compatible))
+        }
```

Comment:
We can merge these blocks to make sure they remain identical.
```suggestion
        "object_safe" | "dyn_compatible" => {
            resolve_property_with(contexts, field_property!(as_trait, is_dyn_compatible))
        }
```